src/Startup/Program.cs

```diff
@@ -101,11 +101,9 @@ public static void Main(string[] args)
 
             using (new Program(args))
             {
-                bool exit = false;
+                var exit = false;
```

Comment:
is that a code style rule? just curious

Reply:
no, but you can easily spot that ```exit``` is a bool ;-)